TLM Config example

articide opened the pull request:
Working configuration for a Tevo Little Monster Delta printer using the MKS Base-L V2.1 Controller board.

Commit:
TLM Config example

Working configuration for a Tevo Little Monster Delta printer using the MKS Base-L V2.1 Controller board.

thinkyhead commented:
These are based on some pretty old configs and would require a major imposition of time and effort for us to bring up to date. Would you mind, please, starting from the configs in `Marlin/src/config/examples/delta/generic` in the current `bugfix-2.0.x` branch and applying the necessary changes to bring these into compliance? Thanks!

Commit:
Updated example TLM config

I updated the TLM config to the latest version using the generic template and ensured it still compiles and works as expexted.

articide commented:
Ok.  I have updated the configs and ensured everything still works.

thinkyhead commented:
Looks good, thanks!
